src/tools/dev/scripts/bv_support/bv_python.sh

```diff
@@ -177,6 +177,26 @@ function bv_python_info
     export MPI4PY_BUILD_DIR=${MPI4PY_BUILD_DIR:-"mpi4py-2.0.0"}
     export MPI4PY_MD5_CHECKSUM="4f7d8126d7367c239fd67615680990e3"
     export MPI4PY_SHA256_CHECKSUM="6543a05851a7aa1e6d165e673d422ba24e45c41e4221f0993fe1e5924a00cb81"
+
```

Comment:
When we want to eventually convert everything to python3, we will already have py3 built here. Other than that, I don't think it will have an effect. I think the first step in that process is just making our current py2 calls compatible with py3.